src/assets/scss/base/_settings.scss

```diff
@@ -0,0 +1,10 @@
+$grey: #232323;
+$pink: #EF597A;
+$yellow: #FFDB22;
+$pink-light: #F8A8BA;
+$white: #FFFFFF;
+
+$h1-font: 'pocket_monkregular';
+$h2-font: 'pocket_monkregular';
+$h3-font: Poppins, sans-serif;
```

Comment:
Perfect 🎉 lets apply them to h1, h2, & h3, below so whenever you add a title, the styling is already done for you.